CHANGES/5572.bugfix

```diff
@@ -0,0 +1 @@
+Always create a new event loop in run_app() to avoid an error caused by using ``asyncio.run()`` beforehand.
```

Comment:
I'm starting to think that this may be a feature, not a bugfix because it's a breaking change.